src/main/kotlin/pt/up/fe/ni/website/backend/service/upload/FileUploader.kt

```diff
@@ -0,0 +1,47 @@
+package pt.up.fe.ni.website.backend.service.upload
```

Comment:
I would only create the `upload/` package if the classes live in 3 different files. In this case, it seems weird

Reply:
I think they should be in different files, what do you think?